### PR TITLE
Fixed code recommendation section

### PR DESCRIPTION
**Pull request by alexeykazakov**

### CODE REVIEW

File: src/app/index.html [hunk 1]
```diff
@@ -270,10 +270,10 @@ <h3 class="bullet-header">
...
                 <h3>
-                  Allow development teams be more confident in the technology decisions they make.
+                  Allow development teams to be more confident in the technology decisions they make.
                 </h3>
                 <p>
-                  OpenShift.io helps developers make better decisions. It provides actionable recommendations and insights around the software components developers use. OpenShift.io Analytics applies machine learning algorithms based on their usage pattern of components. The data is gathered from various public data sources such as Github, Maven and NPM along with our own internal OpenShift data. Through this comprehensive dataset, developers get tangible information for credible
+                  OpenShift.io helps developers make better decisions. It provides actionable recommendations and insights around the software components developers use. OpenShift.io Analytics applies machine learning algorithms based on their usage pattern of components. The data is gathered from various public data sources such as Github, Maven and NPM along with our own internal OpenShift data.
```
Comment (catrobson) [inline src/app/index.html]: Last sentence should be " Through this comprehensive dataset, developers get tangible information for credible decision-making."

Comment (alexeykazakov) [inline src/app/index.html]: Thanks Cat. Fixed.

**Comment by tiwillia** [comment]
LGTM, thanks for fixing this.

**Comment by joshuawilson** [comment]
[test]

**Pull request merged**